docs/time_units.md

```diff
@@ -254,7 +265,7 @@ Mostly, you don't have to convert between the two,
 unless you want to match up information in a tree sequence
 with information written out by SLiM itself.
 In other words, SLiM's time counter measures the number of time steps
-("tick") since the start of the simulation,
+("ticks") since the start of the simulation,
```

Comment:
wondering about the simulation above? would it be better to use the harmonic mean of those gen times?